cmd/simulateEvol/simulateEvol.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/simulate"
+	"log"
+)
+
+func SimulateEvol(rootFastaFile string, treeFile string, simOutFile string, leafOutFile string) {
+	tree := expandedTree.ReadTree(treeFile, rootFastaFile)
+	var fastas []*fasta.Fasta
+	simulate.Simulate(rootFastaFile, tree)
+	nodes := expandedTree.GetTree(tree)
+	fmt.Printf("%s, %s, %s, \n", rootFastaFile, treeFile, simOutFile)
```

Comment:
This is ultimately up to you, but I would consider using log.Printf() here only because I think if someone would try to pipe this to stdout it will be included